webpack.config.js

```diff
@@ -41,15 +44,31 @@ module.exports = {
     // Copy Cesium Assets, Widgets, and Workers to a static directory
     new CopyWebpackPlugin({
       patterns: [
-        { from: path.join(cesiumSource, cesiumWorkers), to: "Workers" },
-        { from: path.join(cesiumSource, "Assets"), to: "Assets" },
-        { from: path.join(cesiumSource, "Widgets"), to: "Widgets" },
-        { from: path.join(cesiumSource, "ThirdParty"), to: "ThirdParty" },
+        {
+          from: path.join(cesiumSource, "Workers"),
+          to: `${cesiumBaseUrl}/Workers`,
+        },
+        {
+          from: path.join(cesiumSource, "ThirdParty"),
+          to: `${cesiumBaseUrl}/ThirdParty`,
+        },
+        {
+          from: path.join(cesiumSource, "Assets"),
+          to: `${cesiumBaseUrl}/Assets`,
+        },
+        {
+          from: path.join(cesiumSource, "Widgets"),
+          to: `${cesiumBaseUrl}/Widgets`,
+        },
+        // If you only installed @cesium/engine use these routes instead:
```

Comment:
IMO it would be cleaner to describe this in the README rather than inline comments.

Reply:
@ggetz I added a section to the readme. Let me know if that looks good